modules/backup-s3/s3/s3.go

```diff
@@ -90,6 +91,8 @@ func (s *s3) GetObject(ctx context.Context, backupID, key string) ([]byte, error
 		return nil, backup.NewErrInternal(errors.Wrapf(err, "get object '%s'", objectName))
 	}
 
```

Comment:
WithLabelValues might panic and we won't to panic because of a metric.
You could use GetMetricWithLabelValues instead

Also might not be a good place to put metrics here since getObject and other methods  are class agnostic.